api/server/request_controller.go

```diff
@@ -0,0 +1,117 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/gin-gonic/gin"
+	"github.com/iron-io/functions/api"
+	"github.com/iron-io/runner/common"
+)
+
+type RequestController interface {
+	AppName() string
```

Comment:
What if they need more information about the App?  Or want to set config/env vars, etc.